java/system-test/src/main/java/sleeper/systemtest/cdk/DeployNewTestInstance.java

```diff
@@ -30,17 +31,26 @@ private DeployNewTestInstance() {
     }
 
     public static void main(String[] args) throws IOException, InterruptedException {
-        if (args.length < 5 || args.length > 6) {
-            throw new IllegalArgumentException("Usage: <scripts-dir> <properties-template> <instance-id> <vpc> <subnet> <optional-deploy-paused-flag>");
+        if (args.length < 5 || args.length > 7) {
+            throw new IllegalArgumentException("Usage: <scripts-dir> <properties-template> <instance-id> <vpc> <subnet> " +
+                    "<optional-deploy-paused-flag> <optional-split-points-file>");
         }
-        DeployNewInstance.builder().scriptsDirectory(Path.of(args[0]))
+        Path scriptsDirectory = Path.of(args[0]);
+        Path splitPointsPath = null;
+        Optional<String> splitPointsPathString = optionalArgument(args, 6);
+        if (splitPointsPathString.isPresent()) {
+            splitPointsPath = Path.of(splitPointsPathString.get());
+        }
+
+        DeployNewInstance.builder().scriptsDirectory(scriptsDirectory)
                 .instancePropertiesTemplate(Path.of(args[1]))
                 .extraInstanceProperties(properties ->
                         properties.setProperty(SYSTEM_TEST_REPO.getPropertyName(), args[2] + "/system-test"))
                 .instanceId(args[2])
                 .vpcId(args[3])
                 .subnetId(args[4])
                 .deployPaused("true".equalsIgnoreCase(optionalArgument(args, 5).orElse("false")))
+                .splitPointsFile(splitPointsPath)
```

Comment:
If this was inlined it might read better with the other arguments. You could make a method to read the argument, and pass in the index, or you could do something like optionalArgument(args, 6).map(Path::of).orElse(null).